app/src/main/res/values/styles.xml

```diff
@@ -341,17 +341,16 @@
 
     <style name="SimpleDraweeViewPlaceholder">
         <item name="actualImageScaleType">centerCrop</item>
-        <item name="placeholderImage">?attr/thumbnail_image_placeholder</item>
         <item name="placeholderImageScaleType">fitCenter</item>
-        <item name="failureImage">?attr/thumbnail_image_placeholder</item>
+        <item name="failureImage">?android:colorBackground</item>
         <item name="failureImageScaleType">fitCenter</item>
         <item name="fadeDuration">0</item>
-        <item name="backgroundImage">?attr/paper_color</item>
+        <item name="backgroundImage">?android:colorBackground</item>
     </style>
 
     <style name="SimpleDraweeViewPlaceholder.SuggestedEdits">
-        <item name="placeholderImage">@drawable/ic_image_gray_transparent_24dp</item>
-        <item name="failureImage">@drawable/ic_image_gray_transparent_24dp</item>
+        <item name="placeholderImage">?attr/material_theme_border_color</item>
```

Comment:
Please see my PR #382  and merge if it looks good to you. If you need a specific placeholder color for Feed cards, you should probably create another style for it.

Reply:
I think Robin wants color-group-21 according to https://www.mediawiki.org/wiki/Wikimedia_Apps/Android_theme_guidelines. setting it @null will not do it. Even for failure image we need to set this attribute color. because, our settings 'show images' will default to failure image styles.

Reply:
Make sense to me.